layer/mounted_layer_windows.go

```diff
@@ -0,0 +1,35 @@
+package layer
+
+import (
+	"io"
+
+	"github.com/docker/docker/pkg/archive"
+)
+
+func (rl *referencedRWLayer) TarStream() (io.ReadCloser, error) {
```

Comment:
Wondering if we should make this error out instead of trying to unmount/mount. Seems dangerous.